styles/all/template/js/collapsiblecategories.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {  // Avoid conflicts with other libraries
+
+	'use strict';
+
+	$(function () {
+		$('a.category[data-id] + .forabg').each(function () {
+			var $this = $(this),
+				id = $this.prev().attr('data-id'),
+				hidden = $this.prev().attr('data-hidden'),
+				$header = $this.find('li.header'),
+				$content = $this.find('.topiclist.forums');
+
+			if (!$header.length || !$content.length) {
+				return;
+			}
+
+			// Add button
+			$header.append('<a class="collapse-btn collapse-' + ((hidden) ? 'show' : 'hide') + '" data-id="' + id + '" data-ajax="phpbb_collapse"></a>');
```

Comment:
Yeah you can get rid of that script tag... we should define it once we have set up our routings